locales-src/onboarding.ftl

```diff
@@ -18,6 +18,12 @@ onboarding-button-label-get-started = Get Started
 onboarding-welcome-header = Welcome to { -brand-short-name }
 onboarding-welcome-body = You’ve got the browser.<br/>Meet the rest of { -brand-product-name }.
 onboarding-welcome-learn-more = Learn more about the benefits.
+onboarding-welcome-form-header = Start Here
+
+onboarding-welcome-modal-variant-a-body = You’ve got the browser.<br/>Now get the most out of { -brand-product-name }.
+onboarding-welcome-modal-variant-b-body = Supercharge your browser’s privacy protection.
+onboarding-welcome-modal-variant-c-body = You’ve got the browser. Let’s add more privacy protection.
+onboarding-welcome-modal-variant-learn-more = Learn about the { -brand-product-name } family of products.
```

Comment:
nit: probably name these more related to the content than the experiment variant as we'll probably delete unused ones eventually:

```ini
onboarding-welcome-modal-get-body = …
onboarding-welcome-modal-supercharge-body = …
onboarding-welcome-modal-privacy-body = …
onboarding-welcome-modal-family-learn-more = …
```

Reply:
Could probably put these directly inline after `onboarding-welcome-body` and `onboarding-welcome-learn-more` respectively to keep them closer to the other alternatives.

Reply:
done